### DISCUSSION TITLE
Conflicts occur between the YVR SDK and the OpenXR's Meta XR Feature.

### DISCUSSION THREAD

**Discussion by minsubb** [PlayForDreamDevelopers/support#11] [Q&A] [answered]
As the YVR SDK is imported into the personal app that used the Meta XR Feature of OpenXR in the order of utilites, core, and platform according to the description and tries to build, a conflict occurs between OVRPrugin's libopenxr_loader.so and libvrplugin's libopenxr_loader.so, and the app does not build normally. 

Also, after installing the YVR SDK, the build using Oculus plugin will not proceed with the same problem.

Is there a solution?

**Comment by YVRDevelopers** [comment, marked as answer]
The conflict is caused by the libopenxr_loader.so, you will need to remove one of the files for it to build.
